time start_time
time end_time
duration speech_duration
time received_time
string transcript
float64 confidence
int8 pid